packages/dashboard-core-plugins/src/index.test.tsx

```diff
@@ -16,16 +18,18 @@ import {
 it('handles mounting and unmount core plugins properly', () => {
   const store = createMockStore();
   render(
-    <Provider store={store}>
-      <Dashboard>
-        <FilterPlugin />
-        <GridPlugin hydrate={() => undefined} />
-        <ChartPlugin hydrate={() => undefined} />
-        <ConsolePlugin />
-        <LinkerPlugin />
-        <MarkdownPlugin />
-        <PandasPlugin hydrate={() => undefined} />
-      </Dashboard>
-    </Provider>
+    <ApiContext.Provider value={dh}>
```

Comment:
@mofojed I'm getting the API in `ChartPlugin` via the `useApi()` hook. What's the right way to test this? Is passing a global via `ApiContext` fine, or what would be a cleaner way?

Reply:
Yes this way of testing is fine. You can just move `@deephaven/jsapi-shim` to the `devDependencies`

Reply:
Moved it `devDependencies` in the Chart package, will do the same in `dashboard-core-plugins` after converting to `useApi`.